### ISSUE TITLE
Suggestion - some tips to add to the wiki

### ISSUE THREAD

**Issue by KatherineCox** [jhudsl/OTTR_Template#365]
A couple of things that have come in handy as we've been working on the AnVIL books, I thought they might be useful to add to the wiki.

<hr>

# Spell check

### URLs

If you want to write out a URL without having to add it to the spell check dictionary, you can style the URL as inline code by adding single back ticks, and it will be ignored by spell check:

Example:

```
Visit [`example.com`](https://www.example.com) now.
```
Visit [`example.com`](https://www.example.com) now.

<hr>

# Formatting tips (new wiki section under Finishing touches?)

## Chapter start

If the big letter at the start of a chapter is causing your text to format oddly, you can add a `<br>` tag to force text to show up below the letter

**Weird formatting**:
```
Short intro text causes weird formatting

1. Item one
1. Item two
1. Item three
```
<img width="461" alt="Screen Shot 2021-12-17 at 1 41 57 PM" src="https://user-images.githubusercontent.com/13210811/146594411-329a722e-e805-4394-9690-fb93ec70f735.png">

**Fixed by adding `<br>` tag**
```
Short intro text causes weird formatting

<br>

1. Item one
1. Item two
1. Item three
```
<img width="430" alt="Screen Shot 2021-12-17 at 1 43 04 PM" src="https://user-images.githubusercontent.com/13210811/146594382-9bc35c9a-2bfd-4b19-988d-c0f165a50b28.png">

**Comment by KatherineCox** [comment]
For formatting tips, there's also the styled boxes, like this:

```
::: {.fyi}
Please click on the subsection headers in the left hand 
navigation bar (e.g., 2.1, 4.3) a second time to expand the 
table of contents and enable the `scroll_highlight` feature 
([see more](introduction.html#scroll-highlight)).
:::
```
<img width="719" alt="Screen Shot 2021-12-17 at 2 10 46 PM" src="https://user-images.githubusercontent.com/13210811/146596142-21dc6868-da1a-4c94-88e1-da4691c93c83.png">

Which work nicely for the bookdown, but I don't know how well they play with other platforms, so I'm not sure if they're recommended

**Comment by cansavvy** [comment]
These are great, @KatherineCox! Thanks for sharing this. We should definitely incorporate these.

**Comment by cansavvy** [comment]
I believe this has been addressed in #372
